src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -184,7 +184,7 @@ export const LeafyGreenSelect: React.VFC<
           disabled={isDisabled}
           value={value}
           {...labelProps}
-          onChange={(v) => onChange(v === "" ? null : v)}
```

Comment:
I don' think it makes sense for the value to ever be `null` because the widget expects a string, which causes the errors